test/Microsoft.Azure.WebJobs.Host.FunctionalTests/WebJobs.Host.FunctionalTests.csproj

```diff
@@ -31,7 +31,7 @@
     <PackageReference Include="Moq" Version="4.16.0" />
     <PackageReference Include="xunit" Version="2.3.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="5.0.0-beta.5" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="5.0.0-beta.3" />
```

Comment:
This will ultimately be updated right? Which prod version will this PR have dependency on for `Microsoft.Azure.WebJobs.Extensions.Storage` ?

Reply:
PROD version will `Microsoft.Azure.WebJobs.Extensions.Storage.5.0.0-beta.3`. This change is just points the tests dll to the future prod version. Referencing to `Microsoft.Azure.WebJobs.Extensions.Storage.5.0.0-beta.5`  does not make change as current version is `Microsoft.Azure.WebJobs.Extensions.Storage.5.0.0-beta.2`